components/proxy/src/test/scala/com/hotels/styx/routing/config/UserConfiguredPipelineFactorySpec.scala

```diff
@@ -44,12 +44,13 @@ import org.scalatest.{FunSpec, ShouldMatchers}
 import rx.lang.scala.Observable
 
 import scala.collection.JavaConverters._
+import scala.collection.JavaConversions._
 
 class UserConfiguredPipelineFactorySpec extends FunSpec with ShouldMatchers with MockitoSugar {
 
   val registries: util.Map[String, StyxService] = ImmutableMap.of("registry1", backendRegistry(newBackendServiceBuilder.path("/foo").build))
-
-  it ("Configures plugins in a given order") {
+  val builtinHandlersFactory = mock[RouteHandlerFactory]
+  ignore ("Configures plugins in a given order") {
```

Comment:
I think we should start adding comments when we ignore tests.

Reply:
Removed the class, and this test too.